### PR TITLE
Refactor home page navigation

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

File: components/avatar/avatarPage.html [hunk 1]
```diff
@@ -56,6 +56,20 @@ <h3 class="header__title">
         <section class="container__sidebar">
             <!-- Drawer Component Regular -->
             <div class="drawer">
+
+                <!-- Getting Started Starts -->
+                <div class="drawer__heading">
+                    <h3 class="drawer__title">Getting Started</h3>
+                </div>
+                <div class="drawer__content-holder" style="margin-bottom: 2rem;">
```
Comment (amartya-rs) [inline components/avatar/avatarPage.html:64]: Any specific reason for the inline style?
Apart from that code looks good to me.

Comment (archit7448) [reply components/avatar/avatarPage.html:64]: +1

**Review by archit7448 — APPROVED** [review]
looks good to me

File: components/avatar/avatarPage.html (hunk 1 above)
Comment (varun13169) [reply components/avatar/avatarPage.html:64]: This part of the documentation will be soon deprecated.

I am working on dynamically populating the sidebar, hence placing the following intermediate code here for submission purposes.

Comment (amartya-rs) [reply components/avatar/avatarPage.html:64]: Cool then, you can go ahead.

**Review by amartya-rs — APPROVED** [review]
Looks good to me.

**Pull request merged**